frontend/src/components/timer/timer.module.css

```diff
@@ -5,11 +5,13 @@
 
 .timer {
```

Comment:
It seems like separate styles for ".red" and ".blue" class is not defined in the CSS. How are you planning to implement this feature? - Let's have a discussion whether separate background colour is necessary for Timer component. According to our Figma, it seems like the timer colour scheme is always black/white. 

If we think adding red/blue will make our app cooler, we can definitely try it out!

Reply:
oh i forget to delete red and blue class, I used them to change the background color in my first timer version. If we want to keep same as our Figma  we can just delete red and blue class. And about time bar, i am waiting for Jason (maybe) to finish then we can merge together